client/css/style.css

```diff
@@ -293,6 +294,16 @@ kbd {
 	content: "\f005"; /* http://fontawesome.io/icon/star/ */
 }
 
+#chat .toggle-button {
+	/* These 2 directives are loosely taken from .fa-fw */
```

Comment:
Nah it is, I updated it from `directly from` to `loosely taken from` :D Just a reminder to keep an eye on `.fw-fa` in the future.